plugins/glustershd/rest.go

```diff
@@ -69,8 +70,11 @@ func selfhealInfoHandler(w http.ResponseWriter, r *http.Request) {
 
 	txn, err := transaction.NewTxnWithLocks(ctx, volname)
 	if err != nil {
-		status, err := restutils.ErrToStatusCode(err)
-		restutils.SendHTTPError(ctx, w, status, err)
+		if err == transaction.ErrLockTimeout {
```

Comment:
Use `ErrToStatusCode` function from `restutils` package. That was the code earlier, as seen in the diff. So, remove your changes.